src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/ReadThreadingAssemblerArgumentCollection.java

```diff
@@ -52,6 +52,14 @@
     @Argument(fullName="padding-around-snps", doc = "Include at least this many bases around an event for calling snps", optional = true)
     public int snpPadding = 20;
 
+    /**
+     * This argument is meant for debugging. Output a -L parsable summary file with all of the boundaries and summary features of each
```

Comment:
Instead of saying -L parsable it would be better to state the specific format -- BED, Picard interval list, etc.